src/generic/caret.cpp

```diff
@@ -74,19 +74,6 @@ int wxCaretBase::GetBlinkTime()
 void wxCaretBase::SetBlinkTime(int milliseconds)
 {
     gs_blinkTime = milliseconds;
-
-#ifdef _WXGTK__
```

Comment:
Is it correct to remove this? It changes wxGTK2 behaviour and I'm not sure why should we do this.

Reply:
As I said in this comment [df3bee9](https://github.com/wxWidgets/wxWidgets/pull/22079/commits/df3bee998062039b7eb5d0223cb6cb938df7252b): the removed code is not executed at all because `_WXGTK__` has a typo: notice the leading underscore, there is only one instead of two.

also we can add support for wxGTK2 easily which I intend to do in a separate PR.

As a side note: the caret under some distros (Ubuntu and its derivatives) consider gtk-cursor-blink-timeout (i.e. time after which the cursor stops blinking, in seconds.)
my question is: should we make our wxCaret blinks in a similar way under wxGTK ?

Reply:
> `_WXGTK__` has a typo

I saw that when it was first checked in. But changing all the cursors is a bad idea, so I just left it there, disabled. And after 8+ years, no one ever noticed that it didn't work, not even the original author. We don't need this, and IMO shouldn't have it.